node-tests/blueprints/helper-test-test.js

```diff
@@ -34,6 +34,26 @@ describe('Blueprint: helper-test', function() {
       });
     });
 
+    describe('with ember-cli-qunit@4.1.1', function() {
+      beforeEach(function() {
+        generateFakePackageManifest('ember-cli-qunit', '4.1.1');
+      });
+
+      it('helper-test foo/bar-baz', function() {
+        return emberGenerateDestroy(['helper-test', 'foo/bar-baz'], _file => {
+          expect(_file('tests/integration/helpers/foo/bar-baz-test.js'))
+            .to.equal(fixture('helper-test/rfc232.js'));
+        });
+      });
+
+      it('helper-test foo/bar-baz --unit', function() {
+        return emberGenerateDestroy(['helper-test', 'foo/bar-baz', '--unit'], _file => {
+          expect(_file('tests/unit/helpers/foo/bar-baz-test.js'))
+            .to.equal(fixture('helper-test/rfc232-unit.js'));
+        });
```

Comment:
I've found the default as `integration` to be confusing for helpers.  But that's probably just me though :man_shrugging:

Reply:
Helpers are only logically used from a template, testing them outside of the template layer should be a fairly uncommon thing to do...